### PR TITLE
readme is more user focused now---ish

### PR DESCRIPTION
**Pull request by DanielShermanSchool**

### CODE REVIEW

**Review by amitcham1 — APPROVED** [review]
Better fits with the idea we need to work for those we want to use this app.

File: README.md [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
 
-MotoReMinder is a universal application that will remind automobile owners when various maintenance intervals are passed based on mileage driven. If the user has access to a Bluetooth enabled OBD-II scanner and their car supports it, the mileage on the car will be automatically updated in the background and push-notifications will be sent to their device to remind them of the intervals coming up.
+### MotoReMinder is a universal application that will remind automobile owners when various maintenance intervals are passed based on mileage driven. If the user has access to a Bluetooth enabled OBD-II scanner and their car supports it, the mileage on the car will be automatically updated in the background and push-notifications will be sent to their device to remind them of the intervals coming up.
 
-To Build: with the flutter SDK installed, clone repo into VSCode or other IDE, choose what virtual device to build onto, and run the default build. (MORE INFORMATION TO BE GIVEN AS NEEDED)
-
-To Run as User: 
-* If on Windows: download windows release, extract to foler, run .exe file
-* If on android: download APK release, install APK
+### Have a car and want to keep track?
```
Comment (jody) [inline README.md:5]: Love this heading!

File: README.md [hunk 2]
```diff
@@ -1,14 +1,14 @@
 # MotoReMinder
 
-MotoReMinder is a universal application that will remind automobile owners when various maintenance intervals are passed based on mileage driven. If the user has access to a Bluetooth enabled OBD-II scanner and their car supports it, the mileage on the car will be automatically updated in the background and push-notifications will be sent to their device to remind them of the intervals coming up.
+### MotoReMinder is a universal application that will remind automobile owners when various maintenance intervals are passed based on mileage driven. If the user has access to a Bluetooth enabled OBD-II scanner and their car supports it, the mileage on the car will be automatically updated in the background and push-notifications will be sent to their device to remind them of the intervals coming up.
```
Comment (jody) [inline README.md:3]: This use of section header format for an entire paragraph seems inappropriate.

Who is your intended reader? Are they necessarily familiar with the phrase "universal application"? This is one case where I think getting a GPT rewrite can be useful by presenting alternative ways to communicate the information to average drivers.

Comment (DanielShermanSchool) [reply README.md:3]: I appreciate the feedback! I'll definitely check out some LLM options and make another pull with an improved section

**Pull request closed without merging**